apps/backend/src/routers/websoc.ts

```diff
@@ -0,0 +1,127 @@
+import { z } from 'zod';
+import type {WebsocAPIResponse, CourseInfo, WebsocCourse} from '@packages/antalmanac-types';
+import { procedure, router } from '../trpc';
+
+function cleanSearchParams(record: Record<string, string>) {
```

Comment:
Should be called `sanitizeSearchParams` so it doesn't sound like a factory method which produces clean search params